Source/Plugins/Core/com.equella.core/js/tsrc/course/EditCourse.tsx

```diff
@@ -197,12 +176,13 @@ class EditCourse extends React.Component<Props, EditCourseState> {
 
   handleSave = () => {
     if (this.props.entity) {
-      const { versionSelection } = this.props.entity;
+      const { versionSelection, students } = this.props.entity;
       const vs = versionSelection === "DEFAULT" ? undefined : versionSelection;
-
+      const st = (students === 0 ? undefined : students);
```

Comment:
Are there cases where the difference between `false`, `0`, `undefined`, and `null` would be significant here?
If not coud
```suggestion
      const st = students || undefined;
```
offer a more concise alternative?

Reply:
Sounds good